image/v1/imagev1.go

```diff
@@ -116,5 +123,11 @@ func rawJSON(value interface{}) *json.RawMessage {
 
 // ValidateID checks whether an ID string is a valid image ID.
 func ValidateID(id string) error {
-	return stringid.ValidateID(id)
+	if len(id) != fullLen {
+		return errors.New("image ID '" + id + "' is invalid")
+	}
```

Comment:
nit: This is redundant as the regex already has `{64}`

Reply:
Hm.. good one; I wonder if it was originally there for performance? I'll leave it for now, but I can benchmark, and if there's no difference, remove it in a follow-up.

Reply:
Could be! I wouldn't bother with benchmarking honestly, just keep it as is. The extra len check doesn't hurt.